packages/data-stores/src/help-center/actions.ts

```diff
@@ -16,37 +14,6 @@ import type {
 } from './types';
 import type { SupportInteraction } from '@automattic/odie-client/src/types';
 
-export const receiveHasSeenWhatsNewModal = ( value: boolean | undefined ) =>
```

Comment:
Looks like we need to adjust some types to get the builds to pass:

https://github.com/Automattic/wp-calypso/blob/b9bac06a4d9649a52ad98b7228f8724b3704a500/packages/data-stores/src/help-center/actions.ts#L225

Reply:
Thanks? How did you find the error?

Reply:
TC shows it in the failed build: https://teamcity.a8c.com/buildConfiguration/calypso_BuildDockerImage/14611117?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&showLog=14611117_1053_756&logView=flowAware